src/projectDB/projectdatabase.cpp

```diff
@@ -1454,9 +1456,9 @@ void ProjectDatabase::deletePeakGroup(PeakGroup* group)
         return;
 
     vector<int> selectedGroups;
-    selectedGroups.push_back(group->groupId);
+    selectedGroups.push_back(group->savedGroupId);
```

Comment:
`groupId` and `savedGroupId` are the same, no? why add two variables for the same.

Reply:
I added `savedGroupId` to store the unique database ID associated with a peak-group that has been saved in an emDB project. I did not want to reuse `groupId` because it is enumerated differently and because we had to get to a working build without disturbing whatever other existing ways `groupId` is currently being used in various parts of the codebase. I agree that, in a restored session both of these values are the same and having only one unique ID associated to the object seems like good software construction. I will make the necessary changes, now that we have some time to refactor. Thanks for pointing out.